pom.xml

```diff
@@ -51,9 +51,6 @@
     <argLine>-Xms768M -Xmx768M -Djava.awt.headless=true -XX:+HeapDumpOnOutOfMemoryError -XX:+TieredCompilation -XX:TieredStopAtLevel=1</argLine>
 
     <jenkins.version>2.204</jenkins.version>
-    <!-- Should only need to override these if using timestamped snapshots https://issues.apache.org/jira/browse/MNG-6274 (but better to use Incrementals instead)  -->
-    <jenkins-core.version>${jenkins.version}</jenkins-core.version>
-    <jenkins-war.version>${jenkins.version}</jenkins-war.version>
     <jenkins-bom.version>${jenkins.version}</jenkins-bom.version>
```

Comment:
Left this since I was not sure what the use case was for overriding `jenkins-bom.version`.